package/gluon-state/Makefile

```diff
@@ -0,0 +1,23 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-state
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/$(PKG_NAME)
+  TITLE:=Provides info about the routers state
+  DEPENDS:=+gluon-core +micrond
+endef
+
+define Package/$(PKG_NAME)/description
+  This script runs every test file in `/lib/gluon/state/check.d` and provides the results as flags in `/var/gluon/state`.
+  It is called by micron.d once every minute.
+  A flag exists in case of its description being true and gets removed otherwise.
+  The flags get opened regardless them being changed.
+
+  This package provides the check `has_default_gw6`, that depends on
+  whether the router has a default IPv6-route on br-client.
```

Comment:
```suggestion
  gluon-state executes checks in `/lib/gluon/state/check.d/` and provides
  a flag file for each check in `/var/gluon/state` depending on the return code
  of the check. A flag file is created (or "touched") if the corresponding check
  exits cleanly and gets removed otherwise. If the flags are "touched", they
  are only accessed, but not modified. This way, the atime of the flag files
  reflect when the last check was performed and the mtime reflects when
  when the state was last changed.

  This package provides the following checks:
  - `has_default_gw6` - check whether the router has a default IPv6-route on br-client.
  
  The checks are executed once every minute (by micron.d).
```

Reply:
Thanks, this sounds way better.